src/universal/styles/rows.ts

```diff
@@ -0,0 +1,13 @@
+// rows.js
+
+import {PALETTE} from 'universal/styles/paletteV2'
+
+export const ROW_BORDER_COLOR = PALETTE.BORDER.LIGHTER
```

Comment:
let's remove this & just let components reference `PALETTE.BORDER.LIGHTER`, that way they don't need to import anything, the value just shows up.